go/oasis-test-runner/oasis/args.go

```diff
@@ -466,6 +466,13 @@ func (args *argBuilder) workerStorageDebugDisableCheckpointSync(disable bool) *a
 	return args
 }
 
+func (args *argBuilder) workerStorageCheckpointSyncEnabled(enable bool) *argBuilder {
```

Comment:
This should be renamed as it doesn't enable checkpoint sync but rather the creation of checkpoints.